src/ansys/fluent/core/install_nltk_data.py

```diff
@@ -0,0 +1,24 @@
+"""Installs NLTK data."""
```

Comment:
@mkundu1 

Now we are able to execute `poetry run install_nltk_data` to install NLTK data but we need to find a way to execute this script during `pip install ansys-fluent-core`.

```
(VirtualEnvs) D:\pyfluent>poetry run install_nltk_data              

D:\Installations\Python310\lib\runpy.py:126: RuntimeWarning: 'nltk.downloader' found in sys.modules after import of package 'nltk', but prior to execution of 'nltk.downloader'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
[nltk_data] Downloading package wordnet to
[nltk_data]     C:\Users\hpohekar\AppData\Roaming\nltk_data...
[nltk_data] Downloading package omw-1.4 to
[nltk_data]     C:\Users\hpohekar\AppData\Roaming\nltk_data...

(VirtualEnvs) D:\pyfluent>
```

Reply:
I don't think this is what we are looking for. As far I read, running custom scripts during installation is not supported in poetry. So for pyfluent, we can download the data on-demand at runtime. For pyconsole, we shall package the data within Ansys Python.

Reply:
Ok. Thank you.